test/Microsoft.AspNetCore.Mvc.FunctionalTests/TempDataPropertyTest.cs

```diff
@@ -25,13 +25,13 @@ public TempDataPropertyTest(MvcTestFixture<BasicWebSite.Startup> fixture)
         public async Task TempDataPropertyAttribute_RetainsTempDataWithView()
         {
             // Arrange
-            var message = "Success (from Temp Data)";
+            var tempDataContent = "Success (from Temp Data)100";
```

Comment:
Why do you need that 100 at the end?

Reply:
Its just a response message that we are trying to verify in the test for the following action:
https://github.com/aspnet/Mvc/pull/7102/files#diff-ed958851cdea5a1bf43528eba0262c0bR57